### PR TITLE
more migration url fixes

### PR DESCRIPTION
**Pull request by integrii**

### CODE REVIEW

**Review by joshulyne — APPROVED** [review]

File: helm-repos/index.yaml [hunk 1]
```diff
@@ -561,17 +491,11 @@ entries:
     maintainers:
     - email: eric.greer@comcast.com
       name: integrii
-    - email: jacob.martin@comcast.com
-      name: lolimjake
-    - email: ianhoegen@gmail.com
-      name: ihoegen
-    - email: jonathan_phan@comcast.com
```
Comment (joshulyne) [inline helm-repos/index.yaml:568]: don't remove!

File: helm-repos/index.yaml [hunk 2]
```diff
@@ -129,17 +113,11 @@ entries:
...
-      name: jonnydawg
```
Comment (joshulyne) [inline helm-repos/index.yaml:137]: dont remove!

File: helm-repos/index.yaml [hunk 3]
```diff
@@ -93,17 +83,11 @@ entries:
...
```
Comment (joshulyne) [inline helm-repos/index.yaml:101]: dont remove!

File: helm-repos/index.yaml [hunk 4]
```diff
@@ -21,17 +21,11 @@ entries:
     maintainers:
     - email: eric.greer@comcast.com
       name: integrii
-    - email: jacob.martin@comcast.com
-      name: lolimjake
-    - email: ianhoegen@gmail.com
-      name: ihoegen
-    - email: jonathan_phan@comcast.com
```
Comment (joshulyne) [inline helm-repos/index.yaml:28]: dont remove!

File: helm-repos/index.yaml [hunk 5]
```diff
@@ -525,17 +461,11 @@ entries:
...
-      name: jonnydawg
```
Comment (joshulyne) [inline helm-repos/index.yaml:533]: dont remove!

File: helm-repos/index.yaml (hunk 2 above)
Comment (integrii) [reply helm-repos/index.yaml:137]: yeah i copied the wrong lines, but its not a big deal...

**Review by jonnydawg — APPROVED** [review]

**Review by jonnydawg — APPROVED** [review]

**Pull request merged**